sections/03_categorize.md

```diff
@@ -290,80 +290,95 @@ by allowing more efficient use of the costly practice of manual annotation.
-deep learning research. EHRs are used differently even within the same health
-care system [@pmid:24159271 @pmid:21347133]. Individual users have unique
-usage patterns, and different departments have different priorities which
-introduce missing data in a non-random fashion. Just et al. demonstrated that
-even the most basic task of matching patients can be challenging due to data
-entry issues [@pmid:27134610]. This is before considering challenges caused by
-system migrations and health care system expansions through acquisitions.
-Replication between hospital systems requires controlling for both these
-systematic biases as well as for population and demographic effects.
-Historically, rules-based algorithms have been popular in EHR-based research but
-because these are developed at a single institution and trained with a specific
-patient population they do not transfer easily to other populations
-[@doi:10.1136/amiajnl-2013-001935]. Wiley et al.
-[@doi:10.1142/9789813207813_0050] showed that warfarin dosing algorithms often
-under perform in African Americans, illustrating that some of these issues are
-unsolved even at a treatment best practices level. Lack of standardization
-makes it challenging for investigators skilled in deep learning to enter the
-field, as numerous data processing steps must be performed before algorithms are
-applied.
-
-Even if data were perfectly standardized, attempts to share data in this domain
-would still encounter technological and legal barriers. A responsibility to
-protect patient privacy limits the ability to  openly share large patient
-datasets. As described above, labeled data are already expensive to obtain. Even
-after they are generated restrictions on sharing can hamper their broad
-distribution. All of these factors combine to result in small samples sizes that
-restrict the number of parameters that can be trained in a model. Though the
-lack of sharing may also hamper reproducibility and physician confidence in
-results, recently described techniques such as Continuous Analysis
-[@doi:10.1038/nbt.3780] can allow reproducible computing on private data. Using
-such techniques intermediate results can be automatically tracked and shared
-without sharing the original data, which may help to address concerns around
-physician confidence.
-
-Raw data isn't the only point of concern in the domain with regards to privacy.
+EHRs are designed chiefly for clinical, administrative and financial purposes, such as patient care, insurance and  billing [@doi:10.1038/nrg3208]. Research is at best a tertiary priority, presenting  significant challenges to EHR-based research in general, and particularly to data intensive deep learning research. These difficulties can be grouped into three areas: local bias, wider standards and legal issues. Note these problems are not restricted to EHR but can also apply to any large biomedical dataset, e.g. clinical trial data.
```

Comment:
This expansion is nice as well. Can you do 80c/line so that reviewers can make comments very close to the text? I think this is better, so I don't have specific comments on it at this time. Someone might in the future though.